src/ethereum_test_tools/spec/blockchain_test.py

```diff
@@ -184,7 +184,7 @@ def make_block(
                 header=header.to_geth_dict(),
                 txs=txs_rlp,
                 ommers=[],
-                withdrawals=to_json_or_none(block.withdrawals),
+                withdrawals=to_json_or_none(env.withdrawals),
```

Comment:
I think this is good, but I think we should also copy the withdrawals from the block into the environment on `block.set_environment` (inside the `set_environment` method), because I think if the block indeed had withdrawals, they would be disregarded at this point.